docs/development/building-from-sources.md

```diff
@@ -36,6 +36,10 @@ These Docker images are also available from the Github packages at
 [`github.com/orgs/pyodide/packages`](https://github.com/orgs/pyodide/packages).
 ```
 
+```{note}
+If `make` is failing with `Permission denied`, you can run `./run_docker` with root permissions: `./run_docker --root`
+```
+
```

Comment:
Sounds good, can you move it down to line 48 after the steps?

Reply:
Done